x-pack/plugins/cases/public/components/owner_context/use_owner_context.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useContext } from 'react';
+import { OwnerContext } from '.';
+
+export const useOwnerContext = () => {
+  const ownerContext = useContext(OwnerContext);
+
+  if (ownerContext.length === 0) {
```

Comment:
Nice error check! One quick thing, if someone tries to use `useOwnerContext` for some reason outside of the scope of the provider, then it will fail on `undefined.length`. I would check if `(!ownerContext || ownerContext.length === 0)`